test/e2e-suite-2/testdata/cycle-backward.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: cycle-backward
+spec:
+  vertices:
+    - name: in
+      source:
+        http: {}
+    - name: cat
+      udf:
+        builtin:
+          name: cat
+    - name: retry
+      replicas: 1
```

Comment:
actually, based on conversation with Derek I need to remove the `replicas` field as it's not part of this spec. My intention was to turn autoscaling off so need to do that the right way